src/ECER.Resources.Documents/Applications/IApplicationRepository.cs

```diff
@@ -1,14 +1,22 @@
-﻿namespace ECER.Resources.Documents.Applications;
+﻿using ECER.Resources.Documents.PortalInvitations;
+
+namespace ECER.Resources.Documents.Applications;
 
 public interface IApplicationRepository
 {
   Task<IEnumerable<Application>> Query(ApplicationQuery query, CancellationToken cancellationToken);
 
+  Task<IEnumerable<Province>> QueryProvinces(ProvinceQuery query, CancellationToken cancellationToken);
```

Comment:
This deserves to go to a metadata resource access service (aka repository interface) as it is not related to an application